data/supportserver/autoyast_supportserver_x86_sle15.xml

```diff
@@ -0,0 +1,229 @@
+      <halt config:type="boolean">false</halt>
+      <second_stage config:type="boolean">true</second_stage>
+    </mode>
+    <proposals config:type="list"/>
+    <!-- The PackageHub repos feature an unknown gpg key. We want to both trust it and import it. -->
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">false</accept_file_without_checksum>
+      <!-- autoyast docu says: ... accept known keys you have not yet trusted . Not enough here. -->
+      <accept_non_trusted_gpg_key config:type="boolean">false</accept_non_trusted_gpg_key>
+      <!-- autoyast docu says: ... new GPG keys of the installation sources... Supposedly needed. -->
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">false</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">false</accept_verification_failed>
+      <!-- autoyast docu says: ... accept and import new GPG keys ... in its database. Supposedly also needed. -->
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage/>
+  </general>
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.suse.de/ibs/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/</media_url>
+        <product>sle-module-basesystem</product>
+        <alias>sle-module-basesystem:15.1::pool</alias>
+        <name>sle-module-basesystem:15.1::pool</name>
+        <priority config:type="integer">99</priority>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <confirm_license config:type="boolean">false</confirm_license>
+        <!-- SLE15SP1 AutoYaST docu sec. 4.9.3 Installing Additional/Customized Packages or Products example 4.24
+             "true" <==> Users are asked whether to add such a product. If so, then
+             selected: defines the default state of pre-selected state of packages -->
+        <ask_user config:type="boolean">false</ask_user>
+        <selected config:type="boolean">true</selected>
+      </listentry>
+      <listentry>
+        <media_url>http://download.suse.de/ibs/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/</media_url>
+        <alias>sle-module-basesystem:15.1::update</alias>
+        <name>sle-module-basesystem:15.1::update</name>
+        <priority config:type="integer">99</priority>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <confirm_license config:type="boolean">false</confirm_license>
+        <ask_user config:type="boolean">false</ask_user>
+        <selected config:type="boolean">true</selected>
+      </listentry>
+
+      <listentry>
+	<!--   NEEDED  not-yet-official update 5.2-5.3.1 for BUGGY tftp-5.2-3.22, bsc#1153625  -->
+        <media_url>http://download.suse.de/ibs/SUSE:/Maintenance:/12899/SUSE_Updates_SLE-Module-Basesystem_15-SP1_x86_64</media_url>
```

Comment:
If I am not mistaken, this is no longer valid (empty repository) as the update is already released.

Reply:
Right.

It wasn't, though, at the time this supportserver was first needed. The absence of this dr*tted bug would have saved me quite some hunting work. Not a fond memory indeed.

Complied with your request with commit 0019204.

Reply:
Could you please mark your change request as addressed? Thanks!